src/main/resources/db/migration/v3__next_migration.sql

```diff
@@ -0,0 +1,7 @@
+--V3__next_migration.sql
+CREATE TABLE auth (
+    id SERIAL PRIMARY KEY,
+    email VARCHAR(255) NOT NULL UNIQUE,
+    password VARCHAR(255) NOT NULL,
+    isactive BOOLEAN DEFAULT FALSE
```

Comment:
Same for this. Please follow previous comments

Reply:
resolved